src/ImageSharp/Image.cs

```diff
@@ -80,8 +82,14 @@ protected Image(Configuration configuration, PixelTypeInfo pixelType, ImageMetad
         /// <inheritdoc />
         public void Dispose()
         {
-            this.Dispose(true);
-            GC.SuppressFinalize(this);
```

Comment:
We shouldn't be removing the finalizer suppression in the base class. 

https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose#the-dispose-method

Reply:
Didn't know `SuppressFinalize` do nothing if type has no finalizer, thanks!